packages/market-service/src/index.ts

```diff
@@ -1,32 +1,43 @@
 import {
-  GetByMarketCapType,
   HistoryData,
+  MarketCapResult,
+  MarketData,
   MarketDataArgs,
-  MarketDataType,
   PriceHistoryArgs,
   PriceHistoryType
 } from '@shapeshiftoss/types'
-import { GetByMarketCapArgs } from '@shapeshiftoss/types/src'
+import { findAllMarketArgs } from '@shapeshiftoss/types/src'
 
-import { MarketService } from './api'
 import { CoinGeckoMarketService } from './coingecko/coingecko'
 
-export const getDefaultMarketService = (): MarketService => {
-  return new CoinGeckoMarketService()
-}
+const MarketProviders = [new CoinGeckoMarketService()]
```

Comment:
add a comment here that the ordering of this array is actually the preference for data sources; most reliable should be first.

Reply:
Done.